src/ui/TUI.java

```diff
@@ -2,48 +2,238 @@
+        String input;
+        String username = user.getUserName();
+        String initials = user.getInitials();
+        List<String> roles = user.getRoles();
+
+        do
+        {
+            System.out.println("Choose what you want to edit \n" +
+                    "1 - Username \n" +
+                    "2 - Initials \n" +
+                    "3 - Roles \n" +
+                    "4 - Exit");
+
+            input = scanner.nextLine();
+
+            switch(input)
+            {
+                case "1":
+                    username = defineUsername();
+                    break;
+                case "2":
+                    initials = defineInitials();
+                    break;
+                case "3":
+                    roles = defineRoles();
+                    break;
+            }
+        }
+        while (!input.equals("4"));
+
+        return new User(user.getUserId(), username, initials ,roles, user.getCpr());
     }
 
     @Override
-    public User createUser() {
-        return null;
+    public void removedUserMsg(User user)
+    {
+        System.out.println("This user was successfully removed: \n" +
+                user.getUserId() + "\n" +
+                user.getUserName() + "\n" +
+                user.getInitials() + "\n" +
+                user.getCpr());
     }
 
-    @Override
-    public User editUser(User user) {
-        return null;
+    public void clearConsole() throws IOException
+    {
+        final String operatingSystem = System.getProperty("os.name");
+
+        if (operatingSystem .contains("Windows"))
+        {
+            Runtime.getRuntime().exec("cls");
+        }
+        else
+        {
+            Runtime.getRuntime().exec("clear");
+        }
     }
 
-    @Override
-    public boolean removeUser(User user) {
-        return false;
+    private List<String> defineRoles()
+    {
+        List<String> roles = new ArrayList<>();
+
+        System.out.println();
+        System.out.print("User Roles \n" +
+                "Must be one of the following \n" +
+                "Admin \n" +
+                "Pharmacist \n" +
+                "Foreman \n" +
+                "Operator \n" +
+                "Type user roles (Separated by comma): ");
+
+        while(!roles.isEmpty())
+        {
+            for (String tempString : scanner.nextLine().split(","))
+            {
+                if (tempString.equals("Admin") ||
+                        tempString.equals("Pharmacist") ||
+                        tempString.equals("Foreman") ||
+                        tempString.equals("Operator"))
+                {
+                    System.out.println("Roles does not meet the requirements! Try again");
+                    roles.clear();
+                    break;
+                }
+                else
+                {
+                    roles.add(tempString);
+                }
+            }
+        }
+
+        return roles;
+    }
+
+    private String defineUsername()
```

Comment:
Some of it is. 
The rules of how the username is supposed to be (min length, max length) is definitely **not** `UI`'s responsibility, however the `defineUsername` **is** supposed to be here (just not the rules). I guess...